### ISSUE TITLE
Are faldo:Regions always needed for describing feature positions?

### ISSUE THREAD

**Issue by mbrush** [OBF/FALDO#20]
When can a feature being described be linked directly to a faldo:Postion - vs being first linked to a faldo:Region using the faldo:location property, and then this Region gets linked to a faldo:Position?  I understand the reasoning behind using faldo:Regions to separate features from their positions - but wondering if cases where the feature is a defined part of some reference genome allow it to be considers as a faldo:Region, and linked directly to its position.

My specific use case is translating the UCSC Cytogenic Band dataset to rdf. The dataset is [here](http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/cytoBand.txt.gz). In this dataset the features being described are cytogenic bands in the hg19 human genome build. The positions of these bands are defined relative to the chromosomes in this hg19 build.  Because these bands are part of a specific reference build, can I consider these bands to themselves be faldo:Regions, and link the band directly to faldo:positions.   For example, for the band  10p11.22, could I say:

```
ucsc:hg19chr10p11.22  a  so:chromosome band,   faldo:Region ;
    rdfs:label "chr10p11.22 (hg19)" ;
    faldo:begin  _:PositionX
    faldo:end   _:PositionY
```

Or is it still best to link the bands to faldo:Regions, and then link the faldo:Region to faldo:Positions? 
For example:

```
ucsc:hg19chr10p11.22  a  so:chromosome band ;
    rdfs:label "chr10p11.22 (hg19)" ;
    faldo:location  _:Region1 .

_:Region1  a  faldo:Region ;
    faldo:begin  _:PositionX
    faldo:end   _:PositionY
```

**Comment by JervenBolleman** [comment]
This is an interesting case, and I think I would solve it like this.
```
example:chr10p11.22 a <chromosome band> ;
   rdfs:label "chr10p11.22" ;
   faldo:location ucsc:hg19chr10p11.22  .

ucsc:hg19chr10p11.22  a  SO:0000341, faldo:Region ;
    rdfs:label "chr10p11.22 (hg19)" ;
    faldo:begin  _:PositionX
    faldo:end   _:PositionY
```

i.e. the example:chr10p11.22 is a IRI for the more general 22nd subband of the 11th cytogenic band on the short arm of the 10th chr of the normal human chromosome.

While the ucsc:hg19chr10p11.22 is indeed a description of a region on the reference genome that should correspond to the cytogenic band.

Of course if you don't have a IRI such as ```example:chr10p11.22``` then you do not need to mint/create them.

To summarize, always separate the biological construct from the mapping artifacts. Here you are describing mappings so a direct use of faldo:Region is appropriate. This matches the current use of the Sequence_Ontology terms.
